src/lib/create_role_list_item.js

```diff
@@ -55,3 +55,27 @@ function createRedirectUri(currentUrl, curRegion, destRegion) {
   }
   return encodeURIComponent(redirectUri);
 }
+
+function createDisplayName(profile, aws_account_id) {
+  const maxLength = 64;
+  const seperator = '  |  ';
+  const overflow = '...';
```

Comment:
Try to use horizontal ellipsis (U+2026) `…` instead of 3 dots.
https://graphemica.com/%E2%80%A6

I tried inputting the character in the switch form manually and it worked fine.